charts/tidb-cluster/templates/scripts/_start_pd.sh.tpl

```diff
@@ -33,7 +33,7 @@ POD_NAME=${POD_NAME:-$HOSTNAME}
 # the general form of variable PEER_SERVICE_NAME is: "<clusterName>-pd-peer"
 cluster_name=`echo ${PEER_SERVICE_NAME} | sed 's/-pd-peer//'`
 domain="${POD_NAME}.${PEER_SERVICE_NAME}.${NAMESPACE}.svc"
-discovery_url="${cluster_name}-discovery.${NAMESPACE}.svc:10261"
+discovery_url="${cluster_name}-discovery.${NAMESPACE}:10261"
```

Comment:
No need to update this as this will cause a rolling update for the users managing with the helm chart.

Reply:
updated